README.md

```diff
@@ -28,54 +40,174 @@ A hotel booking application in React. Homework for the [CodeYourFuture React mod
+**Test:** Verify that when you enter an existing first name or surname and submit the form, the results are filtered accordingly in the customers table.
 
-6. Instead of using your hard-coded data in the `<SearchResults />` component, load data from the `src/data/fakeBookings.json` file in the `<Bookings />` component and pass it as a prop to `<SearchResults />`. Hint: look in the `<Bookings />` component for how to import data from a JSON file.
+#### 20. Highlight only one row when selected
 
-7. Add another column to your `<SearchResults />` table which shows the number of days each booking is staying. Hint: try installing the [moment.js library](http://momentjs.com/) (you'll need to install it with `npm install moment --save`) and using the [`.diff()` method](http://momentjs.com/docs/#/displaying/difference/) to compare dates
+**Instructions:** In the task 16, we highlighted a row when it was clicked. In the following, we will change a bit this logic so only the row which is selected is highlighted (previously, if you clicked on several rows, all of them were highlighted). Instead of recording _if_ a row is clicked or no, we need to record _which_ row is clicked. Declare a new state in the `<SearchResults />` component, which should contain the `id` of the corresponding user when the row is selected. Use this state to highlight the correct row of the table.
 
-### Lesson 2
+**Test:** A row should be highlighted when clicked. When another row is selected, the previous row shouldn't be hightlighted anymore. At a given time, only one row can be highlighted to show which user is selected last.
 
-8. Within `src/App.js`, render the `<Restaurant />` component (that is provided for you in `src/Restaurant.js`) underneath the `<Bookings />` component. Then convert the `<Restaurant />` component to a class component.
+#### 21. Display a customer profile
 
-9. Add a method to the `Restaurant` class named `addOrder`. Use `console.log` to log a "Add order" message. Remember to use the `addOrder = () => {}` syntax. Add a `onClick` handler to the Add `<button>` that calls `this.addOrder`. Ensure that clicking on the button logs your "Add order" message in the console.
+**Instructions:** Create a new `<CustomerProfile />` component. This component should be rendered next to the table in the `<SearchResults />` component. This component can be passed one prop `id`, corresponding to the user id which is selected (as defined in the task 20). Initially, the `<CustomerProfile />` component doesn't show anything. When a customer is selected in the table, fetch the corresponding profile from `https://cyf-react.glitch.me/customers/<ID>` in the `<CustomerProfile />` component. A customer profile should show the customer ID, their email, their nationality, if they are VIP and their phone number.
```

Comment:
We picked this up on Slack, but this will require changes to the [Glitch app](https://glitch.com/~cyf-react) that powers the little backend. I've invited @llh1 to edit that project.

Reply:
I've added the /customers/:id endpoint to the Glitch project, thanks 👍